vision/cloud-client/detect/vision_async_batch_annotate_images.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START vision_async_batch_annotate_images]
+
+from google.cloud import vision_v1
+from google.cloud.vision_v1 import enums
+
+
+def sample_async_batch_annotate_images(
+    input_image_uri="gs://cloud-samples-data/vision/label/wakeupcat.jpg",
+    output_uri="gs://your-bucket/prefix/",
+):
+    """Perform async batch image annotation"""
+    # [START vision_async_batch_annotate_images_core]
+
+    client = vision_v1.ImageAnnotatorClient()
+
+    # input_image_uri = "gs://cloud-samples-data/vision/label/wakeupcat.jpg"
+    # output_uri = "gs://your-bucket/prefix/"
+
+    source = {"image_uri": input_image_uri}
+    image = {"source": source}
+    type_ = enums.Feature.Type.LABEL_DETECTION
+    features_element = {"type": type_}
+    type_2 = enums.Feature.Type.IMAGE_PROPERTIES
+    features_element_2 = {"type": type_2}
```

Comment:
nit: Can we inline these types? Doesn't look like they are being used elsewhere. 

Same with feature_element(s) preferably

Reply:
(same in other samples)

Reply:
Done